tests/benchmark/Tracing/provide-traces-closure.php

```diff
@@ -69,7 +69,8 @@
     Container::DEFAULT_DRY_RUN,
     Container::DEFAULT_GIT_DIFF_FILTER,
     Container::DEFAULT_GIT_DIFF_BASE,
-    Container::DEFAULT_USE_GITHUB_LOGGER
+    Container::DEFAULT_USE_GITHUB_LOGGER,
+    true
```

Comment:
This parameter was missing and crashing my local e2e tests (`make test`).

Something about no-op mutators.